### ISSUE TITLE
getMessage

### ISSUE THREAD

**Issue by tconness** [SymphonyPlatformSolutions/symphony-api-client-node#32]
Hi - great work on node client. I'd like add support for the getMessage v1 api added in v1.53 -- is this repo open to contributors? I wasn't able to push my feature branch. See potential addition to existing MessageClient below:

```
MessagesClient.getMessage = (messageId) => {
  var defer = Q.defer()

  var options = {
    'hostname': SymConfigLoader.SymConfig.agentHost,
    'port': SymConfigLoader.SymConfig.agentPort,
    'path': '/agent/v1/message/' + messageId,
    'method': 'GET',
    'headers': {
      'sessionToken': SymBotAuth.sessionAuthToken,
      'keyManagerToken': SymBotAuth.kmAuthToken
    },
    'agent': SymConfigLoader.SymConfig.proxy
  }

  var req = https.request(options, function(res) {
    var str = ''
    res.on('data', function(chunk) {
      str += chunk
    })
    res.on('end', function() {
      if(SymBotAuth.debug){
        console.log('[DEBUG]', 'MessagesClient/getMessage/str', str)
      }
      defer.resolve(JSON.parse(str))
    })  
  })

  req.end()

  return defer.promise
}
```

**Comment by mistryvinay** [comment]
@tconness I will close this issue as we have now included the above addition via PR #33
https://github.com/SymphonyPlatformSolutions/symphony-api-client-node/pull/33

**Comment by tconness** [comment]
👍 

@mistryvinay Will this be published to npm?
